docs/source/install-plone-restapi.rst

```diff
@@ -0,0 +1,62 @@
+Getting started with plone.restapi
+==================================
+If you haven't installed Plone locally, proceed to the next section, that is, installing Plone with plone.restapi. Make sure you have Python 2.7 and https://virtualenv.pypa.io installed.
+If you have already set-up Plone locally, you can skip the next section and add plone.restapi as an add-on.
+
+Install Plone locally with plone.restapi
+----------------------------------------
+.. code-block::
+
+    git clone git@github.com:plone/plone.restapi
```

Comment:
Isn't `cd plone.restapi` required after this?

Reply:
yup, i'll add it in.